### ISSUE TITLE
[Bug] Super Smash Bros Ultimate basically unplayable in new Ryujinx update

### ISSUE THREAD

**Issue by NicoTheIdiot** [Ryujinx/Ryujinx#5278]
### Description of the issue

In the newest Ryujinx update,Super Smash Bros Ultimate encounters multiple bugs regarding the hitboxes of the players and moving objects. The most noticeable is steve's minecart,that can hit a player randomly even if it's in the complete different side of the map. I also encountered some hitbox bugs like randomly getting hit from attacks that were too far,but couldn't reproduce them. 

### Reproduction steps

Pick steve and fight against any other character and use the minecart move. Most of the times when the minecart goes to the edge of the map the other player gets damaged and knocked back even if obviously not hit. This occurs with different characters too but is mostly noticeable when playing steve.

### Log file

[Ryujinx_1.1.881_2023-06-10_17-08-46.log](https://github.com/Ryujinx/Ryujinx/files/11712704/Ryujinx_1.1.881_2023-06-10_17-08-46.log)

### OS

Windows 10

### Ryujinx version

1.1.881

### Game version

13.0.1

### CPU

i5-12400F

### GPU

NVIDIA GeForce RTX 3060 Ti

### RAM

32 GB

### List of applied mods

ArcCropolis

### Additional context?

This was not a problem 4~ months ago and occurred only after a Ryujinx update. I don't know what exact update it was because I stopped playing for some time and just got back. Also, I just noticed that when shifting,steve's legs can start to flicker and disappear.

**Comment by MutantAura** [comment]
Duplicate of #4731
